apps/prairielearn/src/pages/instructorAssessmentGroups/instructorAssessmentGroups.ts

```diff
@@ -120,9 +120,8 @@ router.post(
     } else if (req.body.__action === 'add_member') {
       const assessment_id = res.locals.assessment.id;
       const group_id = req.body.group_id;
-      const uids = req.body.add_member_uids;
-      const uidlist = uids.split(/[ ,]+/).filter((uid) => !!uid);
-      for (const uid of uidlist) {
+      const uids = parseUidsString(req.body.add_member_uids);
```

Comment:
I think we should consider adding a limit here. Maybe 50 users? I'd rather fail quickly than allow someone to try adding 10 billion users to a group 🙃

Reply:
A safeguard here makes a lot of sense. I think a limit of 50 users makes sense as well. I cannot imagine a scenario of adding more than 50 users to a group at a time.

Reply:
Here yes. In the staff page, large classes may have more than 50 staff, so you may want to consider a larger limit.

Reply:
I could bump it to 100? That said, there are only ~10 courses _globally_ with more than 50 staff members in total, and pretty much all those have just been accumulating staff for the past 3-4 years without removing old permissions, I don't know that anyone has ever tried to add that many users in a single request?

Reply:
I added limits in edc7d8b. I went with 50 UIDs for groups and 100 UIDs for course staff. In cfccaea I displayed the limits to users under the UID inputs.

Reply:
I'm wondering if showing the limits may confuse users, particularly in the groups setting. Groups have an upper limit (though instructors can override it), so showing that the user can add up to 50 names may cause an instructor to get confused about the set up of the assessment.

Reply:
What do you suggest: not show the limit at all, or make the limit take into account the configured group maximum size?

Reply:
The limit is there to avoid situations we don't expect to see in real life. I don't think listing the limit is needed here.

Reply:
Removed in fb46f1a.